subcommands/keys/tuf_show_root.go

```diff
@@ -21,7 +21,16 @@ func init() {
 		Run:   doShowRoot,
 	}
 	show.Flags().BoolVarP(&showProd, "prod", "", false, "Show the production version")
-	cmd.AddCommand(show)
+	tufCmd.AddCommand(show)
+
+	legacyShow := &cobra.Command{
```

Comment:
Maybe it's not worth warning about the command deprecation it at all.

Reply:
It is worth - we're going to remove these legacy commands (already hidden now).

BTW. I reworked the deprecation logic to use the `cmd.Deprecated` attribute; this warrants that the deprecation warning will appear not only in the help message, but also every time the command is invoked by the user.